app/post.go

```diff
@@ -1789,26 +1821,40 @@ func (a *App) countMentionsFromPost(c request.CTX, user *model.User, post *model
 			PerPage:   perPage,
 		})
 		if err != nil {
-			return 0, 0, err
+			return 0, 0, 0, err
 		}
 
+		mentionPostIds := make([]string, 0)
 		for _, postID := range postList.Order {
 			if isPostMention(user, postList.Posts[postID], keywords, postList.Posts, mentionedByThread, checkForCommentMentions) {
 				count += 1
 				if postList.Posts[postID].RootId == "" {
+					mentionPostIds = append(mentionPostIds, postID)
 					countRoot += 1
 				}
 			}
 		}
 
+		if a.isPostPriorityEnabled() {
+			priorityList, nErr := a.Srv().Store().PostPriority().GetForPosts(mentionPostIds)
+			if err != nil {
```

Comment:
@koox00 did you intend to check `nErr` for not nil here?

Reply:
Also, I wonder how the linter missed that.

Reply:
Oh wow, great catch! Yeah the linter sometimes misses things like this.